src/main/webapp/assets/css/product-page.css

```diff
@@ -91,6 +91,24 @@
   color: #fce205;
 }
 
+.star {
+  display: inline-block;
+  position: relative;
+  font-size: 30px;
+  color: #ddd;
+}
+
+.star:after {
+  font-family: Font Awesome 5 Free;
+  content: "\f005";
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 50%;
```

Comment:
Yeah^ it doesn't break anything if i remove the width

the font family problem may be because i did this in global.css:

`  font-family: "Trebuchet MS", Helvetica, sans-serif !important;`

I made it un-overridable because the  fonts were somehow not showing on some pages. 

What is Font Awesome 5 Free font-family for?

Reply:
we use it for the star icon! although if your global.css font family is not overridable then I'm not sure how it works

Reply:
I thought we needed that font for the star icon! Have made the changes